### PR TITLE
Image Classification (ResNet)

### PR DESCRIPTION
**Pull request by amirfaraji**
Hi,

This is for the coding assessment Slava asked me to do 
(build a simple machine learning model and submit your code for review on our open-source platform:
https://produvia.com/ai-platform ).

Thanks,
Amir

### CODE REVIEW

**Comment by slavakurilyak** [comment]
@amirfaraji Well done. Please integrate [MLflow](https://github.com/mlflow/mlflow) into your pull request. MLflow is a code dependency at **ai-platform**. Our [README](https://github.com/produvia/ai-platform/#project-dependencies) file contains more information about this.

File: tasks/computer-vision/image-classification/README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# Image Classification
+# Image Classification
+
+
+Developed an Residual Attention Network where the main branch is based off ResNet18 architecture and the mask branch is a AutoEncoder (U-Net).
```
Comment (lucky-singh) [inline tasks/computer-vision/image-classification/README.md]: please choose one architecture which you give better performance or you can have option to switch between architectures by having parameters. For time being I am reviewing branch which used for pull request.

Comment (lucky-singh) [inline tasks/computer-vision/image-classification/README.md]: Please download Pre-trained model run time. Or add prerequisite steps.

File: tasks/computer-vision/image-classification/README.md [hunk 2]
```diff
@@ -1 +1,10 @@
-# Image Classification
+# Image Classification
+
+
+Developed an Residual Attention Network where the main branch is based off ResNet18 architecture and the mask branch is a AutoEncoder (U-Net).
+
+Tested on MRI image classification.
+
+The Hyperparameters file can be changed for training and to run type:
```
Comment (lucky-singh) [inline tasks/computer-vision/image-classification/README.md]: Please add details and example for training step.

File: tasks/computer-vision/image-classification/README.md [hunk 3]
```diff
@@ -1 +1,10 @@
...
+# Image Classification
+
+
+Developed an Residual Attention Network where the main branch is based off ResNet18 architecture and the mask branch is a AutoEncoder (U-Net).
+
+Tested on MRI image classification.
+
+The Hyperparameters file can be changed for training and to run type:
+
+python main.py
```
Comment (lucky-singh) [inline tasks/computer-vision/image-classification/README.md]: please add `mlflow run .` example. 
Optional: Would it be possible to add more examples or details about the parameters?

**Comment by amirfaraji** [comment]
Hi @lucky-singh, 

I believe I was able to address the issues you mentioned in your review. Parameters, metrics and models are all logged with mlflow. I provided more details about the model, files, parameters, usage and examples in the README file. Finally, the dataset, model and pretrained-weights are downloaded on runtime.

**Comment by slavakurilyak** [comment]
@mlaradji please review this contribution

**Pull request closed without merging**